README.md

```diff
@@ -125,6 +128,42 @@ environment variable to set signing certificate name (e.g Developer ID Applicati
 Assets are saved to `ui/desktop/electron-app/out/make/`.
 DMG packaged desktop UI is available at asset location as `boundary.dmg`.
 
+### Connect Boundary UI to Boundary local instance
+
+This describes how to connect local Boundary UI to your local instance of Boundary.
+
+This assumes you already have up and running a boundary instance in your local environment with a listener `127.0.0.1:9200` (default behaviour). If you do not have it, please [follow Boundary documentation](https://github.com/hashicorp/boundary#build-and-start-boundary-in-dev-mode) for it.
+#### For admin
+
+This assumes you are within `boundary-ui/ui/admin`.
+
+You will need to turn `off` mirage, tell the UI where to find Boundary and run it: `$ ENABLE_MIRAGE=false API_HOST=http://localhost:9200 yarn start`.
+
+Following terminal instructions, open in your browser: `http://localhost:4200/`.
+
+Once you open the UI you will see the login screen.
+
+By default, for development purposes, credentials are:
+- Login name: `admin`.
+- Password: `password`
+
+*Be aware these are default credentials, if you change them in Boundary, use the ones you set before*
+#### For desktop
+
+This assumes you are within `boundary-ui/ui/desktop`.
+
+You will need to turn `off` mirage and run it: `$ ENABLE_MIRAGE=false yarn start:desktop`.
+
+You will see the login screen.
+
+Make sure the origin URL is pointing `http://localhost:9200`.
```

Comment:
I might state this along the lines of "enter the origin of your Boundary dev instance" and mention that the default is localhost:9200.

Reply:
Updated, thanks for the suggestion!